### PR TITLE
gradle updated

### PR DESCRIPTION
**Pull request by nesalang**
added task that upload android aar to maven central

### CODE REVIEW

**Comment by nesalang** [comment]
fixed android-ci issue by updating android ci build defintion

File: source/android/adaptivecards/build.gradle [hunk 1]
```diff
@@ -29,6 +31,65 @@ android {
...
+                        license {
+                            name 'MIT'
+                            url 'https://github.com/Microsoft/AdaptiveCards/blob/master/LICENSE'
+                            distribution 'repo'
+                        }
+                    }
+
+                    developers {
+                        developer {
+                            id 'microsoft'
```
Comment (matthidinger) [inline source/android/adaptivecards/build.gradle]: Does this need to match our groupId from maven?

File: source/android/adaptivecards/build.gradle [hunk 2]
```diff
@@ -29,6 +31,65 @@ android {
...
+                            name 'adaptivecards'
```
Comment (matthidinger) [inline source/android/adaptivecards/build.gradle]: And does this need to match our artifactId?

File: source/android/adaptivecards/build.gradle (hunk 1 above)
Comment (nesalang) [reply source/android/adaptivecards/build.gradle]: no, there is a group id field in the same file; 'setGroupId'

File: source/android/README.md [hunk 3]
```diff
@@ -21,6 +21,10 @@ swig -c++ -java -package com.microsoft.adaptivecards.objectmodel -outdir main/ja
 1. Follow "To compile" section
 2. Build->"Build APK" or "Generate Signed APK..."
 
+# To add adaptivecards library to new project:
+1. add the following to the build.gradle of a project that will use the library
+    compile 'io.adaptivecards:adaptivecards-android-arm:1.0.0'
```
Comment (matthidinger) [inline source/android/README.md]: This should be 0.5.0 i think?

Comment (nesalang) [reply source/android/README.md]: thanks!

**Pull request merged**